src/chrome/content/rules/Blip.xml

```diff
@@ -68,15 +68,9 @@ Fetch error: http://a.blip.tv/ => https://as.blip.tv/: (7, 'Failed to connect to
 
   <securecookie host="^(?:.*\.)?blip\.tv$" name=".+" />
 
-
 	<rule from="^http://a\.blip\.tv/"
 		to="https://as.blip.tv/" />
 
-	<!--	s? for protocol-relative links:
-						-->
-	<rule from="^https?://\d\.i\.blip\.tv/"
```

Comment:
As far as I can tell, `https://d.i.blip.tv/` still times out so this ruleset will cause issues in its current state.

You could either add an exclusion for `d.i.blip.tv` and remove the wildcard for `i.blip.tv` and only list working subdomains.

Reply:
It looks like this ruleset overall has been turned off buy the comprehensive test. Will review this ruleset for deletion.